### PR TITLE
added

### PR DESCRIPTION
**Pull request by avinash367**
Adding my basic document into main

### CODE REVIEW

File: basic [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Git commands
+
+git add . // To add the documents
```
Comment (kulkarni90) [inline basic:3]: is this command to  add single file or multiple file?

File: basic [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+git checkout <branch name> // to checkout the exact branch
+
+
+linux comands
+
+alias clr=clear //to assign a long form of code into short forms
+cat<name> // to see the document inside
+pwd // to see the path of the directory
+mkdr // to make a directory
+chmd // to give the premissions to the github and git
```
Comment (kulkarni90) [inline basic:16]: pls correct this command

**Review by kulkarni90 — COMMENTED** [review]
frame the sentence in proper  ,understanding and meaningfull  way

File: basic (hunk 1 above)
Comment (avinash367) [reply basic:3]: to add all the files

File: basic (hunk 2 above)
Comment (avinash367) [reply basic:16]: okay i will change

File: basic (hunk 1 above)
Comment (kulkarni90) [reply basic:3]: ok  thanks for conformation

**Pull request closed without merging**